### PR TITLE
Add support for prerelease-latest keyword

### PR DESCRIPTION
**Pull request by davidbuzinski**

### CODE REVIEW

File: src/properties.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
     "matlabBatchUrl": "https://ssd.mathworks.com/supportfiles/ci/matlab-batch/v0/install.sh",
     "matlabDepsUrl": "https://ssd.mathworks.com/supportfiles/ci/matlab-deps/v0/install.sh",
-    "matlabLatestReleaseUrl": "https://ssd.mathworks.com/supportfiles/ci/matlab-release/v0/latest",
+    "matlabReleaseInfoUrl": "https://mw-ci-static-dev.s3.amazonaws.com/matlab-release-info/",
```
Comment (mcafaro) [inline src/properties.json:4]: Is this temporary?

**Review by mcafaro — COMMENTED** [review]
Does `prerelease-latest` always stay on prerelease even when the GR comes out?

File: src/properties.json (hunk 1 above)
Comment (davidbuzinski) [reply src/properties.json:4]: Yes, wanted to make sure the integration tests passed. I wanted to try it with the 23b prerelease to make sure the versioning, toolcache, license, etc. worked as expected.

**Comment by davidbuzinski** [comment]
> Does `prerelease-latest` always stay on prerelease even when the GR comes out?

No, maybe `latest-including-prerelease` is better

**Comment by davidbuzinski** [comment]
Reopened as https://github.com/matlab-actions/setup-matlab/pull/87

**Pull request closed without merging**